ui/app/src/components/Header/SearchBar/SearchList.tsx

```diff
@@ -12,10 +12,11 @@
 // limitations under the License.
 
 import { isProjectMetadata, Resource } from '@perses-dev/core';
-import React, { ReactElement, useEffect, useMemo, useState } from 'react';
+import { ReactElement, useEffect, useMemo, useRef, useState } from 'react';
 import { KVSearch, KVSearchConfiguration, KVSearchResult } from '@nexucis/kvsearch';
 import { Box, Button, Chip, Typography } from '@mui/material';
 import Archive from 'mdi-material-ui/Archive';
+import MiddleAlertIcon from 'mdi-material-ui/StarFourPointsOutline';
```

Comment:
Can you change the icon on the home page too for consistency?
<img width="278" height="71" alt="image" src="https://github.com/user-attachments/assets/33c48cfb-264d-4e2a-a64c-7e3277d9993b" />